platform/http_request.cpp

```diff
@@ -121,6 +121,7 @@ class MemoryHttpRequest : public HttpRequest, public IHttpThreadCallback
                     CallbackT onFinish, CallbackT onProgress)
     : HttpRequest(onFinish, onProgress), m_writer(m_downloadedData)
   {
+    m_progress = make_pair(0, 0);
```

Comment:
Это изменение вообще не нужно, Юра. Баг, который нужно исправить, возникает в коде уровнем выше, который неверно интерпретирует second параметр пары.

> On Apr 7, 2016, at 12:19, ygorshenin <notifications@github.com> wrote:
> 
> In platform/http_request.cpp <https://github.com/mapsme/omim/pull/2756#discussion_r58842230>:
> 
> > @@ -121,6 +121,7 @@ class MemoryHttpRequest : public HttpRequest, public IHttpThreadCallback
> >                      CallbackT onFinish, CallbackT onProgress)
> >      : HttpRequest(onFinish, onProgress), m_writer(m_downloadedData)
> >    {
> > +    m_progress = make_pair(0, 0);
> Зачем? Конструктор пары и так все занулит.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2756/files/193bf11a2e6f8a3ead0d909e7f91c3f351ef2e0a#r58842230>